src/main/java/org/generationcp/middleware/domain/germplasm/GermplasmDeleteResponse.java

```diff
@@ -0,0 +1,52 @@
+package org.generationcp.middleware.domain.germplasm;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.Set;
+
+@AutoProperty
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class GermplasmDeleteResponse {
```

Comment:
I think this class makes sense to be placed in BMSAPI since it is used there and not in MW?